### DISCUSSION TITLE
Make canvas transparent?

### DISCUSSION THREAD

**Discussion by belugawhale3** [vinothpandian/react-sketch-canvas#88] [Q&A] [answered]
I would like to overlay the canvas over a three.js 3D scene and allow users to draw on the 3D scene. Currently I'm setting the zIndex to overlay the canvas, but its default color is white so its opaque and blocks the three.js scene.

Would be great to set the canvas to be transparent

**Comment by vinothpandian** [comment, marked as answer]
You can do that by setting the `canvasColor` prop to `transparent`.

Here is an example: https://codesandbox.io/s/transparent-canvas-mpsj9w?file=/src/App.tsx

**Reply by belugawhale3** [reply]
Thanks!
